interface.js

```diff
@@ -0,0 +1,157 @@
+var session_key;
+var user_id;
+
+$(document).ready(function(){
+  var response;
+  var i;
+
+  $.get('https://chitter-backend-api.herokuapp.com/peeps', function(data) {
+    var response = data;
+    for (var i = 0; i < response.length; i++) {
+      var dateTime = response[i].created_at
```

Comment:
The transformation of a single response to html representation could be abstracted into its own function